src/components/ClickableStyle/ClickableStyle.module.css

```
@@ -131,8 +132,11 @@
   background-color: var(--eds-theme-color-button-secondary-neutral-background);
   color: var(--eds-theme-color-button-secondary-neutral-text);
 
-  &:hover,
-  &:focus {
+  svg {
+    color: var(--eds-theme-color-button-icon-neutral-icon);
```

this naming is a bit strange with the double "icon" 🤔  could it be
```suggestion
    color: var(--eds-theme-color-button-icon-neutral);
```
?

---

You got it! I had also forgotten to actually use the hover/focus tokens, so that's in there too now.